### DISCUSSION TITLE
LICENÇA

### DISCUSSION THREAD

**Discussion by mickaelcastilho** [aclap-dev/video-downloadhelper#1255] [License & Purchase requests]
### Your question

Olá, boa noite. Eu usava a minha licença no Google Chrome, mas acabei trocando de computador e agora não consigo mais ter acesso a minha licença, como posso te-lá de volta, nesse mesmo navegador, mas agora no computador que estou usando?

### Addon & CoApp version and details

Comprei minha licença no dia 12 de julho, e usava ela no navegador do Google Chrome. Acabei que tive que trocar o meu computador e agora, usando o mesmo email, eu perdi a licença que eu tinha. Gostaria de recuperar a minha licença e migrá-la para o Firefox, podem me ajudar?

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

You can solve your own problem following instructions that are here on GitHub.  Start by clicking the link to the Wiki.  That link appears at the top of every page of this forum.  That is a sort of user manual for VDH.  You should get in the habit of referring to it regularly.  Once you are there, look for a table of contents down the right side of the page.  Each chapter is a link that you can click.  Over time, you should click every link & read everything you find.  For the moment, click the chapter with the title `FAQ`.  That takes you to a page you should read.  The answer to your problem is on that FAQ page.
